src/dash/Dash.tsx

```diff
@@ -11,7 +13,13 @@ const Dash = () => {
       <Route path={`${match.path}/edit/:articleId`}>
         <EditorPage />
       </Route>
-      <Route path={`${match.path}/`}>
+      <Route path={`${match.path}/issues/:issueId`}>
+        <DashIssueDetail />
+      </Route>
+      <Route path={`${match.path}/issues/`}>
+        <DashIssuesPage />
```

Comment:
With the introduction of these routes, we need to (1) remove the routes in App.tsx, and (2) change the links in the header.

Reply:
I think your mockups have a separate list of routes for dashboard and main site stuff, which I think would be better, since most visitors won't ever see the dashboard.

Reply:
Ahh, I see what you're getting at. My bad.